vtStorAta/DriveEnumeratorAta.cpp

```diff
@@ -64,6 +61,8 @@ eErrorCode cDriveEnumeratorAta::EnumerateDrives( Vector_Drives& AddToList, U32&
             std::shared_ptr<cDriveInterface> drive = std::make_shared<cDriveAta>(std::make_shared<String>(devicePath));
 
             AddToList.push_back( drive );
+            devicePath.empty();
```

Comment:
Yes, I hope it to be fail when checking.
errorCode = GetStorageDeviceHandle( devicePath, deviceHandle );
        if ( eErrorCode::None != errorCode )
        {
            //TODO: handle error
            continue;
        }